pkg/skaffold/deploy/helm/helm.go

```diff
@@ -687,12 +692,9 @@ func sortKeys(m map[string]string) []string {
 
 // getArgs calculates the correct arguments to "helm get"
 func getArgs(v semver.Version, releaseName string, namespace string) []string {
```

Comment:
```suggestion
func getArgs(releaseName string, namespace string) []string {
```
Remove unused parameter

Reply:
thanks, sorry my linters are crashing locally today so i'm having a hard time finding these lol